src/Simulation/Native/src/external/nointrin/kernel1.hpp

```diff
@@ -1,4 +1,4 @@
-// (C) 2018 ETH Zurich, ITP, Thomas Häner and Damian Steiger
+// (C) 2018 ETH Zurich, ITP, Thomas HΣner and Damian Steiger
```

Comment:
unintended change?

Reply:
Yes, this change should be reversed. Something in the git tools keeps doing this; I've had to reverse this change (and other occurrences of `Häner`) several times.